net/netbird/Makefile

```
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netbird
+PKG_VERSION:=0.12.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/netbirdio/netbird/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=c88b65bb9358e5a6f9c34882e77a3414b02d4c5ac13b76fb2e60b952af6a18d7
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/netbirdio/netbird
+GO_PKG_BUILD_PKG:=github.com/netbirdio/netbird/client
+GO_PKG_LDFLAGS_X:=github.com/netbirdio/netbird/client/system.version=$(PKG_VERSION)
+GO_PKG_LDFLAGS:=-s -w
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/netbird
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=VPN
+  TITLE:=Connect your devices into a single secure private WireGuard®-based mesh network
+  URL:=https://netbird.io
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/netbird/description
+  NetBird is an open-source VPN management platform built on top of WireGuard® making it easy to create 
+  secure private networks for your organization or home.
+
+  It requires zero configuration effort leaving behind the hassle of opening ports, complex firewall rules, VPN 
+  gateways, and so forth.
+endef
+
+define Package/netbird/install
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+	$(INSTALL_DIR) $(1)/usr/bin $(1)/etc/init.d
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/client $(1)/usr/bin/
```

```suggestion
	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/client $(1)/usr/bin/netbird
```

or maybe `netbird-client`?

---

No, it's both server and client, so netbird- just my bad that I mistakenly forgot to do it like that- on my test environment it was like that, but I do git pushes from other path and sometimes all stuff isn't in sync.. So suggestion is correct.